proposals/hackygolucky-EXPECTATIONS/EXPECTATIONS.md

```diff
@@ -0,0 +1,108 @@
+## Expectations
+
+With the intent to merge the Node.js Foundation and JS Foundation being announced, those interested in the continued support of Node.js and JS-related projects are working through expectations and structure for what the potential new foundation could look like.
+
+This document is for the initial phase of work for the Bootstrap team to draft requirements that overlap with scope of the Board in order to provide an outline and structure for the foundation attuned to the needs of the projects and ecosystem stakeholders who have participated.
+
+This document is organized by two key areas: concerns that must be included in the foundation bylaws, and those that are communicated expectations that aren't required to be in our legal documents to proceed but are vital to managing expectations of project members and ecosystem stakeholders.
+
+### Bylaws requirements
+- Communicated scope for the foundation and board 
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement 
+- The foundation staff, executive director, and board should follow a leadership style that prioritizes and promotes the well-being of the project and those who prioritize and promote a healthy project and ecosystem. 
+- Project representation on the Board 
+    - Community Representation should sit on the Board directly. Projects could share this representation. 
+    - Community representation on the Board: 3 seats guaranteed for community representation 
+- Board must establish Cross Project Council focused on the specific needs of the member projects 
+    - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board 
+- Non-corporate member representation in budget/finance subcommittee 
```

Comment:
What does non-corporate mean in this context? I think it should be "project representation" instead.

Reply:
I'd agree.  I think what we want is "Allow for easy delegation of budget creation to project members, along with approval decisions against that budget after it has been approved by the board"   This would set us up to be able to have the project take over the approvals/budgeting for as much of the overall budget as make sense.

Reply:
Non-corporate members in bylaws terms for the foundation are project representation in the form of directors from TSC, CommComm, and Individual Membership director. 'Member's in this case is a specific term used and the only non-corporate members that exist are those groups. If we just want to say project(s) representation and define what that actually means elsewhere or later, that's fine by me. I'm not sure from a privacy perspective if there's scenarios where board members, in their fiduciary responsibility, may have to hold private sessions related to money, as corporate members/sponsors often aren't comfortable talking money(this is the sort of thing that can be discussed in the budget/finance meetings) in a public setting, even if the Board members themselves are. So some sensitivity ends up being at play for compromise. Thoughts?

Reply:
It may be 2 different expectations

1) that there is non-corporate representation in the budget/finanance subcomittee
2) that were the project believes it makes sense, creation of the budget and oversight/approval of the spending of that budget against the allocation can be delegated to collaborators in the project.